src/CMakeLists.txt

```diff
@@ -35,3 +35,12 @@ endif()
 
 add_library(cucumber-cpp-nomain STATIC ${CUKE_SOURCES})
 add_library(cucumber-cpp STATIC ${CUKE_SOURCES} main.cpp)
+
+if(WIN32 AND NOT MSVC)
+    target_link_libraries(cucumber-cpp-nomain
+        ws2_32
+        )
+    target_link_libraries(cucumber-cpp
+        ws2_32
+        )
+endif(WIN32 AND NOT MSVC)
```

Comment:
I'd be happy with `if(MINGW)` (if I remember correctly) unless further evidence suggests that it's needed for all `WIN32` compilers that are `NOT MSVC`.